source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,103 @@
+---
+owner_slack: "#modernisation-platform"
+title: Set Up Macie
+last_reviewed_on: 2024-04-18
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+# Introduction
+
+It is relatively easy to set up Macie for users and it need to be done on an account by account basis. **NOTE** Macie ONLY looks at S3 buckets and checks their access information so it will not look at databases or EC2 settings, for example.
+
+To do this you should connect to the environment on which you want to create Macie. Note, for accounts with 4 versions (development, test, preproduction, production) you will need to create Macie on each one where you want to use macie. There is an AWS video explaining the set up and use of macie [demo](https://www.youtube.com/watch?v=8piwEQJJXdo).
+
+AWS also have a brief outline of the purpose of [Macie](https://docs.aws.amazon.com/macie/latest/user/what-is-macie.html) which can be checked to see if this is suitable.
+
+Starting Macie with Terraform is possible and the instructions are shown below. I have also listed the details of setting up jobs. it may be beneficial to set up jobs manually due to the items that can be included. The example only shows a few of the many items that can be included.
+
+# Terraform code
+
+Below there is an example of how the items were set up in both Example and Cooker following the instructions listed here.
+
+## Example Terraform code
+
+There is one **pre-requisite**, for the code shown here, to allow this to work and run successfully. Make sure that the role for **MemberInfrastructureAccess has ace "macie2:*"** set. If this has not been set up correctly (by the Modernisation Platform Team) the terraform listed below will not work. If this is amended in the future, so that it is enabled globally, this comment will be removed.
```

Comment:
should this be ace?

Reply:
Not sure about this comment.
The lines has been left as is

Reply:
Make sure that the role for **MemberInfrastructureAccess has ace "macie2:*"** set. 
are we asking a member to do something here? I wondered whether 'ace' was a typo - as it seems like a strange parameter - but that might be the way it works

Reply:
I didn't notice this at first but it has now been amended. Not entirely sure what this was originally now.